src/index.js

```diff
@@ -13,7 +13,7 @@ function globOptionsWith(compiler, globOptions) {
 }
 
 function getFileDepsMap(compilation) {
-  const fileDepsBy = compilation.fileDependencies.reduce(
+  const fileDepsBy = [...compilation.fileDependencies].reduce(
```

Comment:
@stipsan  why not `Array.from(compilation.fileDependencies)`

which will better indicate that initially `compilation.fileDependencies` is not array

Reply:
First off, performance is better: https://jsperf.com/spread-vs-array-from-set-reduced-to-object

In the current project where I use this plugin there's a couple of thousand of dependencies. Using Array.from is noticeably slower.

According to https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/from it supports turning "array-like" objects while the spread syntax only supports iterators (only arrays, set, map and strings?). This could explain why Array.from is slower. 🤔